src/components/Wishlist/Wishlist.js

```diff
@@ -47,7 +49,28 @@ export const WishlistItems = () => {
                     {product.discount}% Off
                   </span>
                 </div>
-                <button className="add-to-cart">Move to Cart</button>
+                <button
+                  className="add-to-cart"
+                  disabled={
+                    cartState.find((cartItem) => cartItem._id === product._id)
```

Comment:
- Here we can extract this as @rutvikpumak suggested also no need to return boolean value explicitly as its gonna return ```undefined``` if not found and the ```value itself``` if found .
- So we can use undefined for false and value itself for true